ios/Sources/MaaS/UI/Components/InfoButton.swift

```diff
@@ -0,0 +1,127 @@
+import SwiftUI
+import MaasTheme
+
+struct InfoButton: View {
+    
+    public struct InputType {
+        public let text: String
+        public let icon: Image
+        public let foreground: Color?
```

Comment:
Is "foreground" a commonly used term in SwiftUI? Asking because the Android implementation uses the term "content color" - it's fine to diverge if it more closely matches platform conventions, but if not, we should strive for consistency.

Reply:
ForegroundColor is used in SwiftUI, InfoButtonConstants Have been updated multiple times after this input was declared :D will update to "contentColor" to have consistency in all MaasComponents

Reply:
Let's have a discussion next year where it is more important to stick with consistent naming with SwiftUI / Compose respectively vs our own internal naming 🙂 I do think it's valuable to use naming expected by SwiftUI / Compose conventions, especially for components that we expect SDK users to be using directly.